manifests/mod/remoteip.pp

```diff
@@ -0,0 +1,27 @@
+class apache::mod::remoteip (
+  $header            = 'X-Forwarded-For',
+  $proxy_ips         = [ '127.0.0.1' ],
+  $proxies_header    = undef,
+  $trusted_proxy_ips = undef,
+  $apache_version    = $::apache::apache_version
```

Comment:
@igalic @mhaskel I'm not sure to understand why we need to remove apache_version here because in most of the others apache module manifests (worker, itk, etc...) it's used like this.
The main reason is to allow to override the apache version if the auto detection doesn't match with the setup :
example : the auto detection on Debian Wheezy (7.x) returns 2.2 but if you have custom packages with 2.4 version this manifest will not be applied